wiki/content/deploy/fast-data-loading/bulk-loader.md

```diff
@@ -143,6 +143,40 @@ $ dgraph bulk -f <file1.rdf, file2.rdf> ...
 
 ```
 
+### How to properly bulk load
+Starting from Dgraph v20.03.7, v20.07.3 and v20.11.0 depending on your dataset size, you can follow one of the following ways to use bulk loader and start your Cluster.
+
+#### For small dataset
+In case your dataset is small (a few GBs) you can follow these steps:
+1. Run bulk loader only on one server
+2. Once the `out` directory has been created, start the first alpha **only**
+3. Wait for 1 minute to ensure that a snapshot has been taken by the alpha node. By checking the alpha logs, you will be looking for a similar message:
```

Comment:
- Instead of "By checking the alpha logs, you will be looking for a similar message:", "You can confirm that a snapshot has been taken by looking for the following message"